packages/block-library/src/navigation/edit/responsive-wrapper.js

```diff
@@ -102,8 +101,7 @@ export default function ResponsiveWrapper( {
 				>
 					<div { ...dialogProps }>
 						<Button
-							// TODO: Switch to `true` (40px size) if possible
-							__next40pxDefaultSize={ false }
+							__next40pxDefaultSize
```

Comment:
This can be seen, while you add navigation and add the overlay menu icon to mobile or always show, and open the menu by click on overlay icon, you can see the close icon.

Note: For this we have some overriding styles on the icon itself, so it would not have before and after change, but we can see that this change have added our 40px change, so in future if overriding styles is removed, we have default 40px of size.

| Before | After     |
| :-------- | :------- |
| <img width="1429" alt="Resposnsive-close-before" src="https://github.com/user-attachments/assets/5acc7b65-7b47-4896-8ea3-3589477d70ce"> | <img width="1429" alt="Responsive-close-after" src="https://github.com/user-attachments/assets/353b5b00-feba-48fe-8c83-afc3d3e945d1"> |